weather/templatetags/weather_partials.py

```diff
@@ -0,0 +1,312 @@
+from django import template
+from django.utils.translation import gettext_lazy as _
+from django.utils import dateparse
+
+register = template.Library()
+
+# Renders the partial for the daily high/low information
+@register.inclusion_tag("weather/partials/daily-high-low.html")
+def daily_high_low(**kwargs):
```

Comment:
If this is formatted as a pep8 docstring (under the `def`), it will show up in editors like vscode.

I'm also a fan of mandatory docstrings on functions, just because it makes life so much easier when debugging code you haven't touched in a while. (And easier for sysadmins and new devs, too.)